src/lando/ui/jinja2/partials/navbar.html

```diff
@@ -31,23 +31,25 @@
             <p class="control">
               <a class="Navbar-userSettingsBtn button">
                 <span class="icon">
-                  <img src="{{ session['userinfo']['picture']|avatar_url }}" />
+                  <img src="{{request.user.profile.userinfo['picture']|avatar_url }}" />
                 </span>
-                <span>&nbsp;{{ session['userinfo']['name'] }}&nbsp;</span>
+                <span>&nbsp;{{request.user.profile.userinfo['name']}}&nbsp;</span>
```

Comment:
Let's maintain spacing after/before the `{{`/`}}`, like `{{ request...` and then `name' }}`

Reply:
Ah nice catch, yep agreed.